prove/Develop02/entry.cs

```diff
@@ -3,22 +3,21 @@
 using System.IO; 
 
 
-public class Main
+public class Entry
 {
 
-
     Journal Jouranl1 = new Journal();
 
     DateTime date = DateTime.Now;
     
     public void AddEntry()
     {
         // Add List of prompts fr journal entry.
-        Jouranl1._ListOfPrommpts.Add("Who was the most interesting person I interacted with today?");
-        Jouranl1._ListOfPrommpts.Add("What was the best part of my day?");
-        Jouranl1._ListOfPrommpts.Add("How did I see the hand of the Lord in my life today?");
-        Jouranl1._ListOfPrommpts.Add("What was the strongest emotion I felt today?");
-        Jouranl1._ListOfPrommpts.Add("If I had one thing I could do over today, what would it be?");
+        Jouranl1.AddListPrompts("Who was the most interesting person I interacted with today?");
```

Comment:
It seems like you were trying to use this to provide a method to show abstraction, but it doesn't really abstract anything.  The intent is to have a class hold date, prompt, and response information for a single entry.  That allows you to have a few methods to support that structure.  As you can see here, your "abstraction" doesn't really conceal anything since it is a method that does a one-line thing.  This doesn't really provide a simpler interface to conceal the internal complexity since there isn't any internal complexity.